zerver/data_import/mattermost.py

```diff
@@ -132,15 +132,11 @@ def convert_user_data(
     realm_id: int,
     team_name: str,
 ) -> None:
-    user_data_list = []
-    for username in user_data_map:
-        user = user_data_map[username]
-        if check_user_in_team(user, team_name) or user["is_mirror_dummy"]:
-            user_data_list.append(user)
-
-    for raw_item in user_data_list:
-        user = process_user(raw_item, realm_id, team_name, user_id_mapper)
-        user_handler.add_user(user)
+    for raw_item in user_data_map.values():
```

Comment:
This would be easier to read with `user` or `user_data` instead of `raw_item`.

Reply:
switched to `user_data`.